lambdapi.opam

```diff
@@ -19,7 +19,7 @@ depends: [
   "bindlib" {>= "5.0.1"}
   "timed" {>= "1.0"}
   "pratter" {>= "1.1"}
-  "why3" {>= "1.3.1" & <= "1.3.3"}
```

Comment:
The `dune-project` file should be changed, not this one.

Reply:
Do you want me to make the change?

Reply:
If I manage to compile why3, yes

Reply:
Ah sorry, I misread, yes, you can make the change :)